src/components/MergeInfographic/index.tsx

```diff
@@ -1,105 +1,83 @@
 import { useTranslation } from "next-i18next"
-import { AspectRatio, Box, chakra, Icon } from "@chakra-ui/react"
+import type { SVGTextElementAttributes } from "react"
 
 import Translation from "@/components/Translation"
 
+import { cn } from "@/lib/utils/cn"
+
+import { createIconBase } from "../icons/icon-base"
+import { HStack } from "../ui/flex"
+
 import { Background } from "./Background"
 
-const Text = chakra("text", {
-  baseStyle: {
-    textAnchor: "start",
-    fill: "currentColor",
-  },
-})
+const Text = ({
+  className,
+  ...rest
+}: SVGTextElementAttributes<SVGTextElement>) => (
+  <text
+    className={cn("fill-current [text-anchor:start]", className)}
```

Comment:
didn't know that you could do that (arbitrary props) TIL

Reply:
@pettinarip ah yes! Applies to any case where you need to explicitly write a CSS property and value. Of course, you lose the ability to apply a token.

This includes creating CSS variables as well.